### PR TITLE
Review package UI tests

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Frontend tests for view submission [FLA-631](https://justice.gov.bc.ca/jira/browse/FLA-631)

## Type of change

- [x] New test

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

- Tested locally and on github actions

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    public ViewSubmittedPackageSD(AuthenticationPage authenticationPage, PackageReviewPage packageReviewPage) {
+        this.authenticationPage = authenticationPage;
+        this.packageReviewPage = packageReviewPage;
+    }
+
+    @Given("user is on package review page")
+    public void userIsOnPackageReviewPage() {
+
+        String packageReviewPageUrl = MessageFormat.format( "{0}/{1}", packageReviewUrl, "1");
```
Comment (alexjoybc) [inline tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java:35]: Should the package Id be an input parameter ?

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        logger.info("Formatted package review page url:{}", packageReviewPageUrl);
+
+        this.packageReviewPage.goTo(packageReviewPageUrl);
+        this.authenticationPage.signInWithBceid("bobross", "changeme");
```
Comment (alexjoybc) [inline tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java:39]: Could you make those environment variables, I think we will always use a single user to perform all the test, what do you think ?

Comment (karthikmsqa) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java:39]: Thanks Alex, saved it for the next PR, makes sense to update in this one. fixed.

**Review by TayGov — APPROVED** [review]
👍

**Pull request merged**